packages/seacas/libraries/ioss/src/utest/CMakeLists.txt

```diff
@@ -29,8 +29,7 @@ TRIBITS_ADD_EXECUTABLE(
 TRIBITS_ADD_TEST(
         Utst_blob_write_read
 	NAME Utst_blob_write_read
-	NUM_MPI_PROCS ${NPROCS}
-	COMM mpi serial
+	NUM_MPI_PROCS 1
```

Comment:
@gsjaardema, FYI, leaving out the COMM argument is the same as `COMM serial mpi`.